docs_website/versioned_docs/version-4.1.0/api-typedoc.json

```diff
@@ -23625,7 +23625,7 @@
 						},
 						{
 							"id": 2037,
-							"name": "helpType",
+							"name": "helpURL",
```

Comment:
I think we don't need to change this file since it's for v4.1.

Reply:
oops, restored.

Reply:
The entire file needs to be restored.

Reply:
missed. now it is restored and should be identical to the one from dev.